node/src/components/storage/tests.rs

```diff
@@ -378,15 +381,25 @@ fn get_highest_complete_block_header(
     response
 }
 
-/// Stores a block in a storage component.
+/// Stores a block in a storage component and mark it as complete.
 fn put_complete_block(
     harness: &mut ComponentHarness<UnitTestEvent>,
     storage: &mut Storage,
     block: Box<Block>,
 ) -> bool {
+    let block_height = block.height();
     let response = harness.send_request(storage, move |responder| {
-        StorageRequest::PutCompleteBlock { block, responder }.into()
+        StorageRequest::PutBlock { block, responder }.into()
     });
+    if response == true {
+        harness.send_request(storage, move |responder| {
+            BlockCompleteConfirmationRequest {
```

Comment:
Is that something specific to these tests?

Reply:
Not sure if it really needs to mark them complete to be honest. This was changed in https://github.com/casper-network/casper-node/commit/503c4d00fa019f90347a926c2d77c02035e93e3a so I wanted to preserve the same semantics due to that change.